### ISSUE TITLE
Deno build from source failed when compile rusty_v8

### ISSUE THREAD

**Issue by earlymeme** [denoland/deno#7115]
deno: latest
os: win10
toolchain: match all prerequisites

```
error: failed to build archive: bad archive: truncated or malformed archive (off
set to next archive member past the end of the archive after member v8/v8_base_w
ithout_compiler/modules.obj)

error: aborting due to previous error

error: could not compile `rusty_v8`.

Caused by:
  process didn't exit successfully: `rustc --crate-name rusty_v8 --edition=2018
'~\.cargo\registry\src\github.com-1ecc6299db9ec823\rusty_v8-0.8.
1\src\lib.rs' --error-format=json --json=diagnostic-rendered-ansi,artifacts --cr
ate-type lib --emit=dep-info,metadata,link -Cembed-bitcode=no -C debuginfo=2 -C
metadata=71475ee5f70cb60c -C extra-filename=-71475ee5f70cb60c --out-dir 'D:\Work
\learn\deno\target\debug\deps' -L 'dependency=D:\Work\learn\deno\target\debug\de
ps' --extern 'bitflags=D:\Work\learn\deno\target\debug\deps\libbitflags-f5232c5e
063b3cf9.rmeta' --extern 'lazy_static=D:\Work\learn\deno\target\debug\deps\libla
zy_static-c510a28e183e5a75.rmeta' --extern 'libc=D:\Work\learn\deno\target\debug
\deps\liblibc-2ca1af45a67a427c.rmeta' --cap-lints allow -C target-feature=+crt-s
tatic -L 'D:\Work\learn\deno\target\debug\gn_out\obj' -l static=rusty_v8 -l dyli
b=winmm -l dylib=dbghelp` (exit code: 1)
```

**Comment by earlymeme** [comment]
someone give some help?

**Comment by lucacasonato** [comment]
@piscisaureus could you take a look?

**Comment by piscisaureus** [comment]
It looks like librusty_v8 was partially downloaded or somehow got corrupted.

Can you blow out the target/debug/gn_out directory and try to build again?

**Comment by earlymeme** [comment]
@piscisaureus @lucacasonato ok,I will try it, very thanks

**Comment by mash-graz** [comment]
this issue could be simply solved by denoland/rusty_v8#543 -- see: denoland/rusty_v8#545 and #8449
